indico/modules/events/papers/models/call_for_papers.py

```diff
@@ -89,10 +91,19 @@ def layout_reviewers(self):
                 if p.has_management_role('paper_layout_reviewer', explicit=True)}
 
     @property
-    def acl_entries(self):
-        principals = self.managers | self.judges
+    def assignees(self):
+        users = self.judges
```

Comment:
I'd use `set(self.judges)` just to be clear that we don't modify it. Since `judges` returns a new set every time it won't be an issue but I had to look twice to check whether we end up modifying something we don't want to modify below with the in-place changes